resources/views/inputs/upload-image-collection.blade.php

```diff
@@ -37,7 +37,7 @@
     x-on:livewire-upload-error="isUploading = false; livewire.emit('uploadError', '{{ $uploadErrorMessage }}');"
     class="relative z-0 space-y-8"
 >
-    <div class="rounded-xl p-2 border-2 border-dashed border-theme-primary-100 relative {{ $dimensions }}">
+    <div class="rounded-xl p-2 border-2 border-dashed border-theme-primary-100 dark:border-theme-secondary-800 relative {{ $dimensions }} group">
```

Comment:
It's not part of your PR, but when I look at the design in Figma, I think p-2 should be p-1.5